app/controllers/listen.js

```diff
@@ -1,10 +1,48 @@
 import Controller from '@ember/controller';
 import { inject as controller } from '@ember/controller';
 import { inject as service } from '@ember/service';
+import Ember from 'ember';
```

Comment:
imports should come from `@ember/*` mostly, and rarely if ever directly from `Ember`

```
import { computed } from '@ember/object';
```